reporter-web-app/scripts/build.js

```diff
@@ -14,11 +14,13 @@ process.on('unhandledRejection', err => {
 // Ensure environment variables are read.
 require('../config/env');
 
+
```

Comment:
Nit: Blank line added.

Reply:
File comes from upstream create-react-app so prefer not to modify it